### PR TITLE
Part of #17670: Replace Typography with Text component in: callout.js

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation
This PR is a part of #17670 Replace Typography with Text component for:
**ui\components\ui\callout\callout.js**

### Before
Screenshot (69)

### After

Screenshot (70)

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by NidhiKJha — DISMISSED** [review]

File: ui/components/ui/callout/callout.js [hunk 1]
```diff
@@ -38,9 +37,9 @@ export default function Callout({
   return (
     <div className={calloutClassName}>
       <InfoIconInverted severity={severity} />
-      <Typography color={Color.textDefault} className="callout__content">
+      <Text color={Color.textDefault} className="callout__content">
```
Comment (DDDDDanica) [inline ui/components/ui/callout/callout.js:40]:
```suggestion
      <Text color={TextColor.textDefault} className="callout__content">
```

File: ui/components/ui/callout/callout.js [hunk 2]
```diff
@@ -38,9 +37,9 @@ export default function Callout({
   return (
     <div className={calloutClassName}>
       <InfoIconInverted severity={severity} />
-      <Typography color={Color.textDefault} className="callout__content">
+      <Text color={TextColor.textDefault} className="callout__content">
```
Comment (georgewrmarshall) [inline ui/components/ui/callout/callout.js:40]: Need to import `TextColor` and remove `Color` from imports

**Review by georgewrmarshall — COMMENTED** [review]
Need to fix linting issue otherwise LGTM

**Review by georgewrmarshall — DISMISSED** [review]

**Review by DDDDDanica — DISMISSED** [review]

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall ,

I have added the test tokens from the test dApp.
One question / doubt,
these test actions do not cause any actual real charges?

Screenshot (102)

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, just wanted to check in. It looks like this PR is almost there. Did you still want to work on it? If so, that's great! Let us know if you need any assistance or if there are any blockers. If you're unable to continue working on it or haven't had a chance to address it, no worries! I can take it from here and carry it forward. Cheers!

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall ,
I am really sorry for the inconvenience caused.
I was not getting any emails of the comments on my PRs.

Also , due to an unknown issue, I was not able to get the extension running locally, so I was not able to take any screenshots.
Screenshot (105)

I did not want to trouble you.
I tried to find the solution myself, but couldn't.

Now, I am going to fork this again and delete the old fork.
Will that work?

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, looks like you need to run `yarn` to install some missing dependencies. Could you also please re-sign the CLA?

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, thanks for all your effort with this PR I am going to close it as it has become stale. Please feel free to open another PR in future. Thanks for your contributions thus far!

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall ,
Sorry for all the inconvenience caused.
Will try and fix the problem because of which I cannot run it locally.
Have not succeeded yet.
Until then I will not open any new PRs related to this issue as I would not be able to add the screenshots.🙏🏼

Regards,
Harsh

**Comment by PrgrmrHarshShukla** [comment]
I have read and agree to the Contributor License Agreement (CLA).

**Review by georgewrmarshall — APPROVED** [review]
LGTM!

**Review by darkwing — APPROVED** [review]

**Pull request merged**